sale_packaging_price/models/sale_stock.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
Ah OK, I was understanding that you wanted him to rename the model! We agree then :wink: